### ISSUE TITLE
iOS

### ISSUE THREAD

**Issue by maximilize** [ammarahm-ed/react-native-ijkvideo#9]
Do you have any experience with iOS development? We need this library also on iOS, so this is a task I need to take care of soon.

**Comment by ammarahm-ed** [comment]
Yes I have some experience, check my other libraries. And we need to work on iOS to make it work similar to android.

**Comment by marf** [comment]
Hello, we are also looking for this iOS and Android package. It will be really helpful for us, especially the equalizer feature since react-native-video does not support it.
Any plans on when it will be available? 

Keep up the good work!

Thank you
